dev/com.ibm.ws.springboot.fat20.concurrency.app/src/test/java/com/example/demo/ConcurrencyApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package com.example.demo;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class ConcurrencyApplicationTests {
+
+	@Test
+	void contextLoads() throws Exception {
+	}
+
+}
```

Comment:
We don't use `@SpringBootTest` in our builds.  You should remove the `src/test` folder.